pandas/core/frame.py

```diff
@@ -6091,10 +6093,10 @@ def _gotitem(self,
     dtype: float64
     """)
 
-    @Appender(_agg_doc)
-    @Appender(_shared_docs['aggregate'] % dict(
-        versionadded='.. versionadded:: 0.20.0',
-        **_shared_doc_kwargs))
+    @Appender(_shared_docs['aggregate'].format(
+        see_also=_agg_see_also_doc,
+        examples=_agg_examples_doc
+    ) % dict(versionadded='.. versionadded:: 0.20.0', **_shared_doc_kwargs))
```

Comment:
That's a bit too tricky :)

Would be better to use just one of `'..'.format(...)` and `'...' % dict(...)`

But, I think the `@Substitution` does exactly this replacement, I'd use it for consistency with the rest.